system/check_network.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
```

Comment:
stand alone script that can be run to see if the computer is connected to a network and if not and there aren't any nearby that it knows about switch into access point mode